packages/liquidator/src/liquidator.js

```diff
@@ -148,6 +148,210 @@ class Liquidator {
+          from: this.account,
+          gas: Math.min(Math.floor(gasEstimation * this.GAS_LIMIT_BUFFER), this.txnGasLimit),
+          gasPrice: this.gasEstimator.getCurrentFastPrice()
+        };
+
+        // Make sure to keep trying with this nonce
+        const nonce = await this.web3.eth.getTransactionCount(this.account);
+
+        // Min Gas Price, with a max gasPrice of x4
+        const minGasPrice = parseInt(this.gasEstimator.getCurrentFastPrice(), 10);
+        const maxGasPrice = 2 * 3 * minGasPrice;
+
+        // Doubles gasPrice every iteration
+        const gasPriceScalingFunction = ynatm.DOUBLES;
+
+        this.logger.debug({
+          at: "Liquidator",
+          message: "Liquidating position",
+          position: position,
+          inputPrice: scaledPrice.toString(),
+          minLiquidationPrice: this.liquidationMinPrice,
+          maxLiquidationPrice: maxCollateralPerToken.toString(),
+          tokensToLiquidate: tokensToLiquidate.toString(),
+          txnConfig
+        });
+
+        // Receipt without events
+        receipt = await ynatm.send({
+          sendTransactionFunction: gasPrice => liquidation.send({ ...txnConfig, nonce, gasPrice }),
+          minGasPrice,
+          maxGasPrice,
+          gasPriceScalingFunction,
+          delay: 60000 // Tries and doubles gasPrice every minute if tx hasn't gone through
+        });
+      } catch (error) {
+        this.logger.error({
+          at: "Liquidator",
+          message: "Failed to liquidate position🚨",
+          error
+        });
+        continue;
+      }
+
+      const logResult = {
+        tx: receipt && receipt.transactionHash,
+        sponsor: receipt.events.LiquidationCreated.returnValues.sponsor,
+        liquidator: receipt.events.LiquidationCreated.returnValues.liquidator,
+        liquidationId: receipt.events.LiquidationCreated.returnValues.liquidationId,
+        tokensOutstanding: receipt.events.LiquidationCreated.returnValues.tokensOutstanding,
+        lockedCollateral: receipt.events.LiquidationCreated.returnValues.lockedCollateral,
+        liquidatedCollateral: receipt.events.LiquidationCreated.returnValues.liquidatedCollateral
+      };
+
+      // This log level can be overridden by specifying `positionLiquidated` in the `logOverrides`. Otherwise, use info.
+      this.logger[this.logOverrides.positionLiquidated || "info"]({
+        at: "Liquidator",
+        message: "Position has been liquidated!🔫",
+        position: position,
+        inputPrice: scaledPrice.toString(),
+        txnConfig,
+        liquidationResult: logResult
+      });
+    }
+  }
+
+  // Process positions the same way, but pull in tokens from 1inch if balance is not adequate.
+  // This is currently disabled until we can take a second look at it. Its easy to re-enable
+  // by adding this specific call to process positions in the liquidator/index.
+  async liquidatePositionsWithOneInch(maxTokensToLiquidateWei, liquidatorOverridePrice) {
```

Comment:
this is the new function which is identical to the previous liquidatePositions function, just renamed.